version.cmake

```diff
@@ -18,5 +18,5 @@
 # SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
 #
 
-set(ATOMVM_BASE_VERSION "0.6.0-dev")
-set(ATOMVM_DEV TRUE)
+set(ATOMVM_BASE_VERSION "0.6.0-alpha.0")
+set(ATOMVM_DEV FALSE)
```

Comment:
This change will cause every subsequent commit on master to have the same version.  I don't think we want `ATOMVM_DEV` to be `FALSE`

Reply:
We are going to change back to TRUE just after the tag.